app/scripts/filters/findProject.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .filter('findProject', function($filter) {
+    return function(items, searchString) {
+      // return early if no search string
+      if(!searchString) {
+        return items;
+      }
+
+      return _.filter(items, function(item) {
+        var searchStringLower = searchString.toLowerCase();
+
```

Comment:
Unfortunately IE doesn't support [String.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) very well. [_.includes()](https://lodash.com/docs#includes) to the rescue!  There is also a [_.lowerCase()](https://lodash.com/docs#lowerCase) that we might as well use then for a consistent style.